.github/ISSUE_TEMPLATE/stabilization-review.md

```diff
@@ -0,0 +1,50 @@
+---
+name: TC Review Request
+about: Request a Technical-Committee stability review (promotion to Stable/GA)
+title: "TC Review Request: "
+labels: "area/stability"
+assignees: ""
+---
+
+<!--
+Delete the guidance comments after filling in the form.
+-->
+
+# TC Review Request: <Component / Language> <Signal> (API / SDK / Collector-module)
+
+## 1. Promotion scope
+| Signal | Part of project | Current maturity | Target maturity |
+| ------ | -------------- | ---------------- | --------------- |
+| <Tracing \| Metrics \| Logs \| Profiles> | <API / SDK package / Other> | experimental / beta | **stable (GA)** |
+
+## 2. Motivation
+<Why are you requesting GA now? Adoption stats, user demand, risk mitigation, etc.>
+
+## 3. Target version / tag
+`<org>/<repo>@<tag>` – commit `<sha>`
+
+## 4. Links for reviewers
+* **Spec-compliance checklist:** <link>
+* **Compatibility matrix (if any):** <link>
+* **Public docs / getting-started sample:** <link>
+* **CHANGELOG draft:** <link>
+* **Open spec gaps referenced:**
```

Comment:
What does this mean?

Reply:
Added a comment to make it clearer, ptal.